clients/youtube_producer/js/yt_auth/yt_authDirective.js

```diff
@@ -4,23 +4,10 @@ module.exports = function ($window, $http, $interval, $log) {
 		restrict: 'E',
 		link: function(scope, element, attr){
 			//TODO: use more angular.js methods instead of mix of native and angular stuff
-			var userSettings = {
-				userId: ''
-			};
 
-			var fetchPersonalData = (function () {
-				return $http({method: 'GET', url: '/userId'}).then(function (res) {
-					$log.debug('Fetched personal data', res);
-					userSettings.userId = res.data;
-				}).catch(function (error) {
-					$log.error('Error on perosnal data fetch', error);
```

Comment:
the code in red is code that was deleted, so the spelling mistake is already being taken care of.